main.go

```diff
@@ -7,142 +7,150 @@ import (
 	"net/http"
 	"net/url"
 	"os"
+	"reflect"
 	"strings"
+
+	"github.com/asaskevich/govalidator"
 )
 
+func validateUrl(u string) (*url.URL, error) {
+	// Verify the target is HTTP or HTTPS.
+	if !strings.HasPrefix(u, "http://") && !strings.HasPrefix(u, "https://") {
+		return nil, fmt.Errorf("%s: must be an http(s) URL.\n", u)
+	}
```

Comment:
Not sure what this means. Are you saying you *want* trailing spaces after the closing brackets?

(I assumed you meant *newline*, so I added those.)